homeassistant/components/hunterdouglas_powerview/diagnostics.py

```diff
@@ -25,6 +27,7 @@
     DEVICE_MAC_ADDRESS,
     DEVICE_SERIAL_NUMBER,
     PV_HUB_ADDRESS,
+    "configuration_url",
```

Comment:
We have a constant you can use

homeassistant/const.py:ATTR_CONFIGURATION_URL: Final = "configuration_url"